frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -637,7 +637,7 @@ function renderValue(fields, formatOptions, value, options) {
     <ValueComponent
       value={value}
       column={fields[0]}
-      maximumFractionDigits={20}
+      maximum_fraction_digits={20}
```

Comment:
Is there anything we can do to keep using camel-case on the frontend as much as possible?
Very few people would expect a React component accepting props in snake case

Reply:
Hm. The React props are passed straight through to `formatValue` right now. Since the JS wrappers just forward the options, and all the Clojure(Script) logic is behind `metabase.shared.util/normalize-map`, I could expand that to turn both `"snake_case"` and `"camelCase"` into `:kebab-case` and it should Just Work.

More broadly, the options objects for formatting started out as a mix a camelCase and snake_case, with no particular pattern. (I suspect JSON convention in `visualization_settings` fields leads to snake_case, and React props convention to camelCase.) This porting effort has started out standardizing on snake_case but I think it could be swung the other way if we wanted. @ranquild ?

Reply:
I think `camelCase` makes sense sense TBH. Let's just pick something and stick with it. @bshepherdson, we could use https://github.com/clj-commons/camel-snake-kebab to handle the `camelCase` => `kebab-case` and back transformations

Reply:
Okay, I'm using `camel-snake-kebab` as suggested. Now the Clojure code will see uniform `:kebab-case-keywords` whatever the FE code passes as the keys of `options`. I've switched the React prop uses back to camelCase.

The only place that's a bit wonky is in the type for the options, but most of the call sites aren't TS right now, so we can sidestep the problem for now.